posts/2021-03-24-microprofile40-open-liberty-21003.adoc

```diff
@@ -0,0 +1,442 @@
+---
+layout: post
+title: "MicroProfile 4.0 support in Open Liberty 21.0.0.3"
```

Comment:
This title is too similar to GA release post and I'd like to be a little more descriptive about the fact that this blog is taking a closer look at the APIs.

Possible alternative suggestions:
"A Deep Dive into MicroProfile 4.0 with Open Liberty"
"Making the most of MicroProfile 4.0 with Open Liberty"
"Taking a deeper look at what's new in MicroProfile 4.0 with Open Liberty"
"What's new in MicroProfile 4.0 with Open Liberty"